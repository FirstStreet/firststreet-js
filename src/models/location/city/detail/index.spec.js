const City = require('./index');
const cityMock = require('../../../../__mocks__/cityDetail');

let city = null;
describe('City', () => {
  beforeEach(() => {
    city = new City(cityMock);
  });

  it('should initialize', () => {
    expect(city).toMatchSnapshot();
  });
});
