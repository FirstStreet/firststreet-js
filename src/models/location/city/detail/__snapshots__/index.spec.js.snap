// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`City should initialize 1`] = `
LocationCityDetail {
  "_data": Object {
    "county": Array [
      Object {
        "fsid": 12071,
        "name": "Lee County",
      },
    ],
    "fsid": 1263700,
    "geometry": Object {
      "bbox": Object {
        "coordinates": Array [
          Array [
            Array [
              -82.186855,
              26.414299,
            ],
            Array [
              -82.005686,
              26.414299,
            ],
            Array [
              -82.005686,
              26.500827,
            ],
            Array [
              -82.186855,
              26.500827,
            ],
            Array [
              -82.186855,
              26.414299,
            ],
          ],
        ],
        "type": "Polygon",
      },
      "center": Object {
        "coordinates": Array [
          -82.099848254,
          26.453150971,
        ],
        "type": "Point",
      },
      "polygon": Object {
        "coordinates": Array [
          Array [
            Array [
              Array [
                -82.184887,
                26.478207,
              ],
              Array [
                -82.186855,
                26.470189,
              ],
              Array [
                -82.17809,
                26.462136,
              ],
              Array [
                -82.173874,
                26.459211,
              ],
              Array [
                -82.17039,
                26.457697,
              ],
              Array [
                -82.164316,
                26.455112,
              ],
              Array [
                -82.153881,
                26.449797,
              ],
              Array [
                -82.148718,
                26.445776,
              ],
              Array [
                -82.147995,
                26.444761,
              ],
              Array [
                -82.143073,
                26.437859,
              ],
              Array [
                -82.13732,
                26.432861,
              ],
              Array [
                -82.132175,
                26.430202,
              ],
              Array [
                -82.124816,
                26.42737,
              ],
              Array [
                -82.112509,
                26.422635,
              ],
              Array [
                -82.099463,
                26.417616,
              ],
              Array [
                -82.08928,
                26.415072,
              ],
              Array [
                -82.082652,
                26.414299,
              ],
              Array [
                -82.080515,
                26.414386,
              ],
              Array [
                -82.074362,
                26.414638,
              ],
              Array [
                -82.067823,
                26.415735,
              ],
              Array [
                -82.062874,
                26.417396,
              ],
              Array [
                -82.04876,
                26.423348,
              ],
              Array [
                -82.039576,
                26.428101,
              ],
              Array [
                -82.037452,
                26.4292,
              ],
              Array [
                -82.017699,
                26.440955,
              ],
              Array [
                -82.014125,
                26.443082,
              ],
              Array [
                -82.00858,
                26.446382,
              ],
              Array [
                -82.005686,
                26.451578,
              ],
              Array [
                -82.005782,
                26.453378,
              ],
              Array [
                -82.005859,
                26.454824,
              ],
              Array [
                -82.005865,
                26.45493,
              ],
              Array [
                -82.008151,
                26.458455,
              ],
              Array [
                -82.010903,
                26.460426,
              ],
              Array [
                -82.015512,
                26.460704,
              ],
              Array [
                -82.019181,
                26.459978,
              ],
              Array [
                -82.021336,
                26.459551,
              ],
              Array [
                -82.022285,
                26.459408,
              ],
              Array [
                -82.023637,
                26.459203,
              ],
              Array [
                -82.027732,
                26.458585,
              ],
              Array [
                -82.029965,
                26.458821,
              ],
              Array [
                -82.032216,
                26.459444,
              ],
              Array [
                -82.033109,
                26.459988,
              ],
              Array [
                -82.035445,
                26.454557,
              ],
              Array [
                -82.035484,
                26.454493,
              ],
              Array [
                -82.035883,
                26.453831,
              ],
              Array [
                -82.035981,
                26.453974,
              ],
              Array [
                -82.036036,
                26.454106,
              ],
              Array [
                -82.03611,
                26.454387,
              ],
              Array [
                -82.036165,
                26.45447,
              ],
              Array [
                -82.036283,
                26.454517,
              ],
              Array [
                -82.036228,
                26.454869,
              ],
              Array [
                -82.036125,
                26.455167,
              ],
              Array [
                -82.034251,
                26.459003,
              ],
              Array [
                -82.033331,
                26.460947,
              ],
              Array [
                -82.034049,
                26.461225,
              ],
              Array [
                -82.038215,
                26.464161,
              ],
              Array [
                -82.041574,
                26.466395,
              ],
              Array [
                -82.045437,
                26.469723,
              ],
              Array [
                -82.055437,
                26.475585,
              ],
              Array [
                -82.059564,
                26.476456,
              ],
              Array [
                -82.060904,
                26.476496,
              ],
              Array [
                -82.061302,
                26.476507,
              ],
              Array [
                -82.062659,
                26.475784,
              ],
              Array [
                -82.065732,
                26.47369,
              ],
              Array [
                -82.068233,
                26.472117,
              ],
              Array [
                -82.069663,
                26.471651,
              ],
              Array [
                -82.07479,
                26.472697,
              ],
              Array [
                -82.076737,
                26.472356,
              ],
              Array [
                -82.082401,
                26.470365,
              ],
              Array [
                -82.08944,
                26.463653,
              ],
              Array [
                -82.089226,
                26.463398,
              ],
              Array [
                -82.089655,
                26.463006,
              ],
              Array [
                -82.090298,
                26.462807,
              ],
              Array [
                -82.091173,
                26.463056,
              ],
              Array [
                -82.095836,
                26.467911,
              ],
              Array [
                -82.104036,
                26.472732,
              ],
              Array [
                -82.111771,
                26.476059,
              ],
              Array [
                -82.114914,
                26.47741,
              ],
              Array [
                -82.11606,
                26.477903,
              ],
              Array [
                -82.119597,
                26.478771,
              ],
              Array [
                -82.124814,
                26.482791,
              ],
              Array [
                -82.126136,
                26.48381,
              ],
              Array [
                -82.126725,
                26.484514,
              ],
              Array [
                -82.128601,
                26.484753,
              ],
              Array [
                -82.13255,
                26.48465,
              ],
              Array [
                -82.138089,
                26.483628,
              ],
              Array [
                -82.142126,
                26.483524,
              ],
              Array [
                -82.145788,
                26.48381,
              ],
              Array [
                -82.147468,
                26.484503,
              ],
              Array [
                -82.150076,
                26.48667,
              ],
              Array [
                -82.153881,
                26.491277,
              ],
              Array [
                -82.156436,
                26.493961,
              ],
              Array [
                -82.15774,
                26.494915,
              ],
              Array [
                -82.165494,
                26.498579,
              ],
              Array [
                -82.170335,
                26.500144,
              ],
              Array [
                -82.172568,
                26.500356,
              ],
              Array [
                -82.173015,
                26.500827,
              ],
              Array [
                -82.173301,
                26.500356,
              ],
              Array [
                -82.174655,
                26.498446,
              ],
              Array [
                -82.174728,
                26.498469,
              ],
              Array [
                -82.176116,
                26.495657,
              ],
              Array [
                -82.176209,
                26.495436,
              ],
              Array [
                -82.176285,
                26.495255,
              ],
              Array [
                -82.176386,
                26.495013,
              ],
              Array [
                -82.176488,
                26.494771,
              ],
              Array [
                -82.176802,
                26.494025,
              ],
              Array [
                -82.177682,
                26.492237,
              ],
              Array [
                -82.178139,
                26.491308,
              ],
              Array [
                -82.180182,
                26.487158,
              ],
              Array [
                -82.182417,
                26.482752,
              ],
              Array [
                -82.182773,
                26.482097,
              ],
              Array [
                -82.182968,
                26.481739,
              ],
              Array [
                -82.184887,
                26.478207,
              ],
            ],
          ],
        ],
        "type": "MultiPolygon",
      },
    },
    "lsad": "25",
    "name": "Sanibel",
    "neighborhood": Array [],
    "state": Object {
      "fsid": 12,
      "name": "Florida",
    },
    "zcta": Array [
      Object {
        "fsid": 33957,
        "name": "33957",
      },
    ],
  },
}
`;
